LICENSE.md

```diff
@@ -52,39 +52,36 @@ The User accepts the following liability:
  - This Mod may alter fundamental parts of the Game.
  - The User is liable for all damages resulting from the use or misuse of the Mod.
 
-3. Right to use
+3. Right to use and examine
```

Comment:
I'm curious why you want to merge these two sections. In the overall license, no other section combines two areas of rights.